src/benchmarks/reconstruction/tracking_efficiency/CMakeLists.txt

```diff
@@ -31,4 +31,4 @@ plugin_include_directories(${PLUGIN_NAME}
 
```

Comment:
Ok, instead of putting it to each factory, lets put it in global plugin_add macro then

Somewhere around

https://github.com/eic/EICrecon/blob/main/cmake/jana_plugin.cmake#L31

Shouldn't be too bad.